src/briefcase/platforms/linux/appimage.py

```diff
@@ -259,16 +259,33 @@ def run_app(self, app: AppConfig, **kwargs):
         """
         self.logger.info("Starting app...", prefix=app.app_name)
         try:
-            # Start streaming logs for the app.
-            self.logger.info("=" * 75)
-            self.tools.subprocess.run(
+            # Start the app in a way that lets us stream the logs
+            log_popen = self.tools.subprocess.Popen(
                 [os.fsdecode(self.binary_path(app))],
-                check=True,
                 cwd=self.tools.home_path,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.STDOUT,
+                bufsize=1,
             )
+
+            try:
+                # Start streaming logs for the app.
+                self.logger.info(
+                    "Following log output (type CTRL-C to stop log)...",
```

Comment:
This also appears to be adding back in the "send CTRL-C to exit" messages that were [removed](https://github.com/beeware/briefcase/pull/921#issuecomment-1281159124) in #921.

Reply:
Oh yeah - I forgot we made a conscious decision to pull those out.